### PR TITLE
[cilium-health] Fix ip v6 return when is enabled.

### PR DESCRIPTION
**Pull request by yanggangtony**
Signed-off-by: yanggang <gang.yang@daocloud.io>

/kind cleanup

Fix ipv6 return when it is enabled. otherwise return ''..

### CODE REVIEW

File: pkg/health/server/node.go [hunk 1]
```diff
@@ -23,7 +23,12 @@ func (n *healthNode) PrimaryIP() string {
 	if n.NodeElement.PrimaryAddress.IPV4.Enabled {
 		return n.NodeElement.PrimaryAddress.IPV4.IP
 	}
-	return n.NodeElement.PrimaryAddress.IPV6.IP
+
+	if n.NodeElement.PrimaryAddress.IPV6.Enabled {
+		return n.NodeElement.PrimaryAddress.IPV6.IP
+	}
+	return ""
```
Comment (joestringer) [inline pkg/health/server/node.go:30]: What is the difference?

**Comment by joestringer** [comment]
Does this functionally change something, for example preventing a nil dereference, or is this code cleanup?

**Comment by yanggangtony** [comment]
@joestringer 
may be the later , code cleans.

**Comment by joestringer** [comment]
This patch doesn't make a difference to the access patterns of the structures in a way that could impact users, and the code style is not informed by any current linter. If you have a linter that is highlighting this particular change, then I suggest that you propose adding that linter to the codebase.

Style changes without a robot to enforce them just introduce churn in the tree without pushing the project forward, so it's better that we don't spend our time adjusting code style in PRs like this.

**Pull request closed without merging**